examples/cloudwatch_alert_for_ecs_service/main.tf

```diff
@@ -20,25 +20,30 @@ module "terra3_examples" {
   # dependency: required for downloading container images
   nat = "NAT_INSTANCES"
 
-  # Choose the metric type to perform auto scaling based on.
-  #metric_type = "CPU_UTILISATION" #default
-  #metric_type = "MEMORY_UTILISATION"
+  # Choose the metric type to perform CloudWatch alert.
+  cpu_utilization_alert    = false
+  memory_utilization_alert = false
 
-  # CloudWatch CPU and MEMORY configurations
+  # CloudWatch alert CPU Utilization configurations
   cpu_utilization_high_evaluation_periods = 3
   cpu_utilization_high_period             = 300
   cpu_utilization_high_threshold          = 90
   cpu_utilization_low_evaluation_periods  = 3
   cpu_utilization_low_period              = 300
   cpu_utilization_low_threshold           = 20
 
+  # CloudWatch alert MEMORY Utilization configurations
   memory_utilization_high_evaluation_periods = 3
   memory_utilization_high_period             = 300
   memory_utilization_high_threshold          = 90
   memory_utilization_low_evaluation_periods  = 3
   memory_utilization_low_period              = 300
   memory_utilization_low_threshold           = 20
 
+  # SNS subscription endpoint list.
+  # No email should be provided when CPU/Memory utilisation is disabled.
+  endpoint_email = []
```

Comment:
Rename it to sth like alert_receivers_email to better understand the meaning of the attribute at first sight.

Reply:
renamed.